sympy/core/function.py

```diff
@@ -3231,11 +3230,7 @@ def count_ops(expr, visual=False):
                 # count the args
                 ops.append(o*(len(a.args) - 1))
             elif a.args and (
-                    a.is_Pow or
-                    a.is_Function or
-                    isinstance(a, Derivative) or
-                    isinstance(a, Integral) or
-                    isinstance(a, Sum)):
+                    a.is_Pow or a.is_Function or isinstance(a, (Derivative, Integral, Sum))):
```

Comment:
is_Pow is the same as isinstance(Pow) so this could all be combined into a single isinstance call.

Reply:
Is this really true in all cases? Include user subclassing? Didn't want this PR to be a functional change so I left it as is.

Reply:
I don't think we need to worry about user subclassing but I agree that keeping a distinction between stylistic and semantic changes in different PRs is good.